feel/feelmodels/modelmodels.hpp

```diff
@@ -0,0 +1,69 @@
+/* -*- mode: c++; coding: utf-8; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; show-trailing-whitespace: t  -*-
+
+ This file is part of the Feel++ library
+
+ Author(s): Vincent Chabannes <vincent.chabannes@feelpp.org>
+ Date: 9 Feb 2018
+
+ Copyright (C) 2018 Feel++ Consortium
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2.1 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library; if not, write to the Free Software
+ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+#ifndef FEELPP_MODELMODELS_HPP
+#define FEELPP_MODELMODELS_HPP 1
+
+#include <boost/property_tree/ptree.hpp>
+#include <feel/feelcore/feel.hpp>
+
+namespace Feel {
+
+namespace pt = boost::property_tree;
+
+class FEELPP_EXPORT ModelModel
```

Comment:
add documentation to the class for Doxygen and understand what it is doing. 
The name seems quite a bit awkward